packages/universal-router/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@shopify/universal-router",
+  "version": "0.0.0",
```

Comment:
Let's keep this sub-1.0 until we have our own thing, and make sure the docs are clear about that

Reply:
yeah I'll add a note/banner to the readme.